### PR TITLE
JCLOUDS-1497: Fix checkstyle-suppressions for jcloud-labs

### PR DESCRIPTION
**Pull request by oflebbe**
This is the jclouds part of fixing checkstyle. jclouds-labs has a couple of violations as well

### CODE REVIEW

**Comment by nacx** [comment]
Closing and re-opening to trigger the build again

File: resources/checkstyle.xml [hunk 1]
```diff
@@ -48,7 +48,7 @@
         </module>
         <module name="NoWhitespaceBefore"/>
         <module name="RedundantImport"/>
-        <module name="RedundantModifier"/>
+       <!-- <module name="RedundantModifier"/> -->
```
Comment (nacx) [inline resources/checkstyle.xml]: The PR LGTM, but why are you removing this check?

Comment (oflebbe) [reply resources/checkstyle.xml]: We have some static imports (search for "import static") in the code. I am doubting it is worth the effort to fix it. Wdyt ?

Comment (nacx) [reply resources/checkstyle.xml]: I'm wondering why this check was not failing before. Anyway, I agree there is little value in this one.

Comment (gaul) [reply resources/checkstyle.xml]: This check should find redundant modifiers like `public` in interfaces, not static imports.

Comment (nacx) [reply resources/checkstyle.xml]: Right. I ran a local build and only in `core` there were more than 40 violations. I fixed them, then found the same in another project... So I agree the value this check provides is close to zero and I think removing it is OK.

Comment (oflebbe) [reply resources/checkstyle.xml]: I run the report again, and yeah sorry you are right: The violations where way down the file not on top where the "import static" was used.

**Pull request merged**